app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_counter
+    user.increment!(:comments_counter)
+  end
```

Comment:
- [ ] Comment model should have a method that updates the comments counter for a post, in your case it is user 😉 :
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```